### DISCUSSION TITLE
Reuse the library in WinUI3 project

### DISCUSSION THREAD

**Discussion by jasmeet21** [lepoco/wpfui#387] [Q&A] [answered]
Hi [pomianowski](https://github.com/pomianowski), would like to check with you if there is a way to re-use this library with WinUI3 project, would there be any limitation ?
The reason being WinUI3 does not have some items/controls like your library does e.g ( store navigation / snackbar and etc ). I have not use WinUI3 but planning to migrate my project if your library supports that framework.

**Comment by pomianowski** [comment, marked as answer]
This project only works for **WPF**. There are no plans yet to create another library for _Windows App SDK_ or _UWP_.
